packages/anoncreds/src/models/registry.ts

```diff
@@ -37,7 +37,9 @@ export interface AnonCredsRevocationRegistryDefinition {
 export interface AnonCredsRevocationStatusList {
   issuerId: string
   revRegDefId: string
-  revocationList: number[]
+  revocationList: Array<number>
   currentAccumulator: string
+  issued?: Array<number>
+  revoked?: Array<number>
```

Comment:
```suggestion
```

These should not be added here. The revocation status list is the model as used by AnonCreds now, and defined in the AnonCreds specification. Indy uses deltas and in those deltas are `issued` and `revoked` keys. 

So we need to do the following:
- Whenever we want to issue/revoke we create or update a status list, then in the indy anoncreds registry implementation we take the current status list and compare it to the new status list. Based on that we can get the issued/revoked indices
- Whenever we want to get a status list (this is already implemented before your PR) we get the detlas, and based on all deltas we construct the revocation list.

So it's important to know: Indy ALWAYS uses detlas on ledger, but everywhere else we use revocation status list as much as possible. So the indy way of doing things should only be used in the Indy anonCreds registry and be transformed to the anoncreds public models  (which is revocation status list)

Reply:
How do we construct a status list from the deltas? Is that something that indy-vdr provides?

Reply:
There's an util for that in AFJ :  https://github.com/openwallet-foundation/agent-framework-javascript/blob/1d333770dcc9e261446b43b5f4cd5626fa7ac4a7/packages/indy-vdr/src/anoncreds/utils/transform.ts#L3C7-L3C7